plat/nvidia/tegra/common/tegra_sip_calls.c

```diff
@@ -87,6 +89,18 @@ uint64_t tegra_sip_handler(uint32_t smc_fid,
 			SMC_RET1(handle, -ENOTSUP);
 		}
 
+		/*
+		 * The GPU is the user of the Video Memory region. In order to
+		 * transition to the new memory region smoothly, we program the
+		 * new base/size ONLY if the GPU is in reset mode.
+		 */
+		regval = mmio_read_32(TEGRA_CAR_RESET_BASE +
+				      TEGRA_GPU_RESET_REG_OFFSET);
+		if ((regval & GPU_RESET_BIT) == 0UL) {
```

Comment:
Was this meant to be U or UL? 
UL is also used in other commits / files; if using U is more appropriate should update those too.